api/src/main/java/org/openmrs/api/impl/ObsServiceImpl.java

```diff
@@ -173,6 +165,18 @@ private Obs saveExistingObs(Obs obs, String changeMessage) {
 		return newObs;
 
 	}
+        
+        private void unsetVoidedAndCreationProperties(Obs newObs,Obs obs) {
+                // unset any voided properties on the new obs
```

Comment:
@PKatGITHUB please remove the comments in this method. the method name is now long and meaningful enough that the comments are not needed.